manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -410,6 +412,11 @@ spec:
       name: lightstep-certs
       readOnly: true
     {{- end }}
+    {{ range .Values.global.proxy.hybrid.secretVolumes }}
```

Comment:
I don't think this is the right path forward, should be exposed over SDS

See also https://github.com/istio/istio/issues/32202; the main problem is auth

Reply:
thanks @howardjohn , I'll fix this one

Also, should I move this PR to `master` as well, similar to the [API one](https://github.com/istio/api/pull/2165)?

Reply:
Yes, new code is only accepted on `master`. (sometimes) you can change it in the github UI by clicking at the top where it says release-1.12 without creating a new PR

Reply:
hey @howardjohn , there are just way too many conflicts after changing the branch. I'll just go ahead and create a new PR and link this PR for historical reference.